homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -319,13 +319,7 @@ async def _async_force_resync(self, *_: Any) -> None:
         """Force a resync after an update since the hub may have stale state."""
         self._forced_resync = None
         _LOGGER.debug("Force resync of shade %s", self.name)
-        await self._async_force_refresh_state()
-
-    async def _async_force_refresh_state(self) -> None:
-        """Refresh the cover state and force the device cache to be bypassed."""
-        await self._shade.refresh()
-        self._async_update_shade_data(self._shade.raw_data)
-        self.async_write_ha_state()
+        await self.async_update()
```

Comment:
You'll still need to write state after the update here because the call isn't coming through the service.

Reply:
I'd probably leave this one as-is unless you wanted to replace the two lines here:
```python3
        await self._shade.refresh()
        self._async_update_shade_data(self._shade.raw_data)
```

with `async_update()` which would make it a bit more try.